src/components/Auth/useAuthProvider.ts

```diff
@@ -1,54 +1,34 @@
-import { useMutation, UseMutationOptions } from '@tanstack/react-query'
 import { useClient } from '@vocdoni/react-providers'
```

Comment:
@selankon As a general recomendation, repeated in the comments later, what ever is not strictly login and logout, but amore generic reusable function should be probably moved in aseparate folder

Reply:
Are you agree to take this changes on this PR https://github.com/vocdoni/ui-scaffold/pull/769 to merge this now?